change/@fluentui-web-components-ec172ca8-23f3-4d01-a969-6504fa67a87a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33023&commitId=aa88013961b5ae2dc58dd97bb3d2a1d01c3046d1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|

</details>


  <details><summary><b>Pivot</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Count and icon.chromium.png|149| Changed|
|Pivot.Large.chromium.png|149| Changed|
|Pivot.Count And Icon RTL.chromium.png|149| Changed|
|Pivot.Root.chromium.png|149| Changed|

</details>


  <details><summary><b>Pivot - Overflow</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.chromium.png|424| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|2| Changed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Place Holder Dark Mode.chromium.png|2| Changed|
|react-charting-SankeyChart.Basic.chromium.png|4| Changed|
|react-charting-SankeyChart.Place Holder RTL.chromium.png|3| Changed|
|react-charting-SankeyChart.Basic RTL.chromium.png|4| Changed|
|react-charting-SankeyChart.Basic Dark Mode.chromium.png|2| Changed|
|react-charting-SankeyChart.PlaceHolder.chromium.png|3| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>